gum/arch-arm/gumthumbrelocator.c

```diff
@@ -153,6 +162,21 @@ gum_thumb_relocator_increment_outpos (GumThumbRelocator * self)
   g_assert (self->outpos <= self->inpos);
 }
 
+inline guint8 
+get_it_instrument_scope (guint16 code) 
```

Comment:
Function name is not namespaced as per the style guide. Should also be `static`. The `inline` keyword should not be used unless profiling indicates that it makes a difference.